osbenchmark/metrics.py

```diff
@@ -102,13 +93,7 @@ def refresh(self, index):
         return self.guarded(self._client.indices.refresh, index=index)
 
     def bulk_index(self, index, doc_type, items):
-        # TODO #653: Remove version-specific support for metrics stores before 7.0.0.
-        # pylint: disable=import-outside-toplevel
-        import opensearchpy.helpers
-        if self._cluster_version[0] > 6:
-            self.guarded(opensearchpy.helpers.bulk, self._client, items, index=index, chunk_size=5000)
-        else:
-            self.guarded(opensearchpy.helpers.bulk, self._client, items, index=index, doc_type=doc_type, chunk_size=5000)
```

Comment:
Can you add some context on why removing metric store support for ES < 6  is needed here?

Reply:
Removing support for ES 6 and below is not explicitly needed but would require an additional maintenance burden for us. ES 7 deprecated certain parameters related to put template operations (more info [here](https://www.elastic.co/guide/en/elasticsearch/reference/7.17/removal-of-types.html)). If we want to continue supporting ES 6 as a data store we would need to create duplicates of the index templates and add some additional branching logic.